src/main/java/picard/TestStringUtilSplit.java

```diff
@@ -6,20 +6,21 @@
 import java.util.StringTokenizer;
 
 /**
- *
+ * A class that can be used to check which is faster, StringUtil.split or StringTokenizer.
+ * Currently, StringUtil is faster.
  */
-public class Test {
+public class TestStringUtilSplit {
     private static final String TEXT = "C0A69ACXX111213:6:1101:10000:144257\t83\t5\t128984606\t60\t76M\t=\t128984542\t-140\tAGTGTTAGAACTTCCTCCCCAAAGCATATACTTCAGTGGCAAGCTGTCCTGGATGAAGGTATGACCAACCAGATCA\t@FFFEECC>EFHBJIGIFGIEIJJJIHED<IEHIGIIJIIIGJIGJIIIIIJGGCJIIGIHHHBHGHFFDFFFC@@\tXT:A:U\tNM:i:0\tSM:i:37\tAM:i:37\tX0:i:1\tX1:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tMD:Z:76";
 
     public static void main(String[] args) {
-        new Test().run();
+        new TestStringUtilSplit().run();
     }
 
-    public void run() {
+    public double run() {
         final int ITERATIONS = 1000000;
         final String[] fields = new String[10000];
         final StopWatch watch = new StopWatch();
-
+        final long stringUtilTime;
         watch.start();
         for (int i=0; i<ITERATIONS; ++i) {
```

Comment:
Should have spaces around operators, e.g. `(int i = 0; i < ITERATIONS; ++i)`. Using command-option-L in intellij should fix them all.